From c70e637bbcb99878f2c101d2868a5da8ec6c0647 Mon Sep 17 00:00:00 2001
From: sabrinaharris <sabrina.harris@digital.cabinet-office.gov.uk>
Date: Mon, 21 Dec 2020 12:08:44 +0000
Subject: [PATCH] Update fixes date for Tech Docs Template issues

---
 source/accessibility.html.md.erb | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/source/accessibility.html.md.erb b/source/accessibility.html.md.erb
index 4c6b9d9..8c63ca0 100644
--- a/source/accessibility.html.md.erb
+++ b/source/accessibility.html.md.erb
@@ -1,6 +1,6 @@
 ---
 title: Accessibility statement for GovWifi technical documentation
-last_reviewed_on: 2020-09-21
+last_reviewed_on: 2020-12-21
 review_in: 6 months
 hide_in_navigation: true
 ---
@@ -75,10 +75,10 @@ We used manual and automated tests to look for issues such as:
 
 ## What we’re doing to improve accessibility
 
-We plan to fix the issues with the Technical Documentation Template by the end of 2020.
+We plan to fix the issues with the Technical Documentation Template by the end of March 2021.
 
 ## Preparation of this accessibility statement
 
-This statement was prepared on 18 September 2020. It was last reviewed on 21 September 2020.
+This statement was prepared on 18 September 2020. It was last reviewed on 21 December 2020.
 
 This website was last tested in August 2020. The test was carried out by the technical writing team at GDS. We used the [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org/) and a checklist we created with the help of the GDS accessibility team. We tested a sample of pages from each section of the site.
-- 
2.24.3 (Apple Git-128)

